.github/workflows/deploy-to-control-plane.yml

```
@@ -0,0 +1,181 @@
+name: Deploy to Control Plane
+
+run-name: ${{ github.event_name == 'pull_request' || (github.event_name == 'issue_comment' && github.event.issue.pull_request) ? 'Deploying Review App' : format('Deploying {0} to Staging App', github.ref_name) }}
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 3: mapping values are not allowed in this context

(syntax-check)